d3-dsv is not in the dependency list for @dodona/dolos-lib

```bash
node index.js 
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module 'd3-dsv'
Require stack:
- /home/fabian/dev/tmp/dolos-test/node_modules/@dodona/dolos-lib/dist/dolos.js
- /home/fabian/dev/tmp/dolos-test/node_modules/@dodona/dolos-lib/dist/index.js
- /home/fabian/dev/tmp/dolos-test/index.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:999:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/home/fabian/dev/tmp/dolos-test/node_modules/@dodona/dolos-lib/dist/dolos.js:34:18)
    at Module._compile (node:internal/modules/cjs/loader:1097:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1149:10)
    at Module.load (node:internal/modules/cjs/loader:975:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:999:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/fabian/dev/tmp/dolos-test/node_modules/@dodona/dolos-lib/dist/dolos.js',
    '/home/fabian/dev/tmp/dolos-test/node_modules/@dodona/dolos-lib/dist/index.js',
    '/home/fabian/dev/tmp/dolos-test/index.js'
  ]
}

Node.js v17.1.0
```

--- a/cli/package.json
+++ b/cli/package.json
@@ -24,17 +24,12 @@
     "access": "public"
   },
   "devDependencies": {
-    "@types/benchmark": "^2.1.0",
-    "@types/d3": "^6.7.0",
     "@types/express": "^4.17.12",
     "@types/node": "^16.9.0",
     "@typescript-eslint/eslint-plugin": "^4.33.0",
     "@typescript-eslint/parser": "^4.33.0",
-    "ava": "^3.8.2",
-    "benchmark": "^2.1.4",
     "eslint": "^7.27.0",
     "np": "^7.5.0",
-    "nyc": "^15.0.1",
     "ts-node": "^10.2.1",
     "typescript": "^4.5.5"
   },
@@ -46,7 +41,6 @@
     "cliui": "^7.0.4",
     "commander": "^8.1.0",
     "csv-stringify": "^6.1.0",
-    "d3": "^6.7.0",
     "express": "^4.17.1",
     "open": "^8.2.1",
     "tree-sitter": "^0.20.0",

--- a/lib/package.json
+++ b/lib/package.json
@@ -25,6 +25,8 @@
     "access": "public"
   },
   "devDependencies": {
+    "@types/benchmark": "^2.1.1",
+    "@types/d3-dsv": "^2.0.0",
     "@types/node": "^16.9.0",
     "@typescript-eslint/eslint-plugin": "^4.29.2",
     "@typescript-eslint/parser": "^4.29.2",
@@ -37,8 +39,10 @@
     "typescript": "^4.5.5"
   },
   "dependencies": {
+    "d3-dsv": "^2.0.0"
   },
   "optionalDependencies": {
+    "@elm-tooling/tree-sitter-elm": "^5.5.1",
     "tree-sitter": "^0.20.0",
     "tree-sitter-bash": "^0.19.0",
     "tree-sitter-c": "^0.20.1",

--- a/lib/src/test/dolos.test.ts
+++ b/lib/src/test/dolos.test.ts
@@ -196,7 +196,7 @@ test("should read CSV-files", async t => {
 
   const report = await dolos.analyzePaths(["../samples/javascript/info.csv"]);
 
-  t.is(4, report.files().length)
+  t.is(4, report.files().length);
 
   t.is(5, report.scoredPairs.length);
   const { similarity } = report.scoredPairs[0];

--- a/lib/src/test/dolos.test.ts
+++ b/lib/src/test/dolos.test.ts
@@ -190,3 +190,15 @@ test("changed order should be a good match", async t => {
   t.is(4, pair.fragments().length);
 
 });
+
+test("should read CSV-files", async t => {
+  const dolos = new Dolos();
+
+  const report = await dolos.analyzePaths(["../samples/javascript/info.csv"]);
+
+  t.is(4, report.files().length)
+
+  t.is(5, report.scoredPairs.length);
+  const { similarity } = report.scoredPairs[0];
+  t.true(similarity > 0.75);
+});

--- a/samples/javascript/info.csv
+++ b/samples/javascript/info.csv
@@ -1,0 +1,5 @@
+filename,id,labels,created_at
+sample.js,1,original,2019-07-23 17:12:33 +0200
+copy_of_sample.js,2,copy,2019-07-25 11:02:57 +0200
+copied_function.js,3,copy,2019-07-25 14:43:20 +0200
+another_copied_function.js,4,copy,2019-07-27 19:22:39 +0200

